CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### `Added`
 
+* [#40](https://github.com/nf-core/eager/issues/40) - Added the pileupCaller genotyper from sequenceTools
```

Comment:
As the major upgrades are on top - can we shift this down the list (right behind the major additions?). Would be nicer I think 👍